bin/melody

```diff
@@ -1,13 +1,19 @@
 #!/usr/bin/env php
 <?php
 
-if (!file_exists(__DIR__.'/../vendor/autoload.php')) {
-    echo "Melody's dependencies not installed.\n";
+if (file_exists($autoloader_path = __DIR__.'/../../../autoload.php')) {
+    require_once $autoloader_path;
+} else if (file_exists($autoloader_path = __DIR__.'/../vendor/autoload.php')) {
```

Comment:
It should be `elseif`.

Then I would try the `$autoloader_path = __DIR__.'/../vendor/autoload.php'` first, because it's going to be hit much more often, and so it saves one call.